src/main/java/org/ibp/api/java/impl/middleware/dataset/validator/DatasetValidator.java

```diff
@@ -232,4 +235,33 @@ public void validateVariableBelongsToVariableType(final int datasetId, final int
 			throw new ApiRequestValidationException(this.errors.getAllErrors());
 		}
 	}
+
+	public void validateNotDeletingSystemEntryDetailToAlreadyGeneratedExperiment(final Integer datasetId,
```

Comment:
@lik-83 just a suggestion, I would use Removing instead of Deleting, same in messages

Reply:
Done!